understanding/20/on-input.html

```diff
@@ -42,7 +42,22 @@ <h2>Intent of On Input</h2>
          
       </div>
       
-      
+      <div class="note">
+
+         <p>If the user interface component is not fully operable with the keyboard due to the <a class="glossary">change of context</a>, 
+		    this would be a violation of SC 2.1.1.
+         </p>
+
+      </div>
+
+      <div class="note">
+
+         <p>A <a class="glossary">change of context</a> should only be made if this makes further operation of the application easier. 
+		 If the change of context makes the operation difficult for impaired people, it is recommended to avoid the context change. 
```

Comment:
is "impaired people" an appropriate term?

Reply:
yes that's a rather unfortunate turn of phrase